### PR TITLE
Started the guide

### PR DESCRIPTION
**Pull request by karinakozarova**
Updated the readme with little information and what should be added in the future.

### CODE REVIEW

**Comment by nelsonic** [comment]
@karinakozarova this is looking good so far! please _assign_ the Pull Request for review when you want someone to review and merge. 👍

**Comment by iteles** [comment]
@karinakozarova A little tip from someone who has been looking into project histories a lot lately - if you could add a little more detail to your commit messages, that will be super useful for developers who come after you and are looking for specific things. 

For the ones above, I would suggest commit messages along something like this:
+ `Adds introduction on what is HTML to readme #3`
+ `Adds HTML basics #3`
+ `Adds HTML syntax #3`

No need to change anything of course but just wanted to give you a tip for the future! Developers in later years will love you ;)

**Comment by karinakozarova** [comment]
Hello 😄 I started the guide. Can you give me some tips on what to adjust? And if anything looks really bad? Thanks in advance.

**Comment by karinakozarova** [comment]
@iteles  @nelsonic maybe time to merge so that others can start contributing too?

**Review by nelsonic — APPROVED** [review]
@karinakozarova this is a _really good_ start! 🎉

**Comment by iteles** [comment]
@karinakozarova So sorry I didn't spot this earlier! Will do a little reviewing asap and get it merged in 🙌

File: README.md [hunk 1]
```diff
@@ -1,2 +1,346 @@
-# lean-html5
-Learn how to use HTML5 (from scratch) to build websites/web applications!
+# Learn HTML5
+Learn how to use HTML5 (from scratch) to build websites/web applications! No previous knowledge needed.
+
+## Some history
```
Comment (iteles) [inline README.md:4]: @karinakozarova We like to [start our tutorials with why](https://www.ted.com/talks/simon_sinek_how_great_leaders_inspire_action), but this isn't a blocker and I love that you have a little history in here. In a future iteration we can add it in 👍

File: README.md [hunk 2]
```diff
@@ -1,2 +1,346 @@
...
+
+## What is HTML?
+HTML (short for **H**yper**T**ext **M**arkup **L**anguage) is basically 
+everything you see on the web everyday - every website uses HTML
+(including GitHub, Google, Facebook, your aunt's blog, etc).
+HTML is just the structure of the page - the texts, images, 
+buttons **but** it doesn't include the beautifiers for your page(CSS, BootStrap)
+although they can be inserted into the html file itself.
+
+## So HTML isn't pretty?
```
Comment (iteles) [inline README.md:15]: :+1:

File: README.md [hunk 3]
```diff
@@ -1,2 +1,346 @@
...
+Plain HTML isn't but it can be made awesome looking using styles and libraries(CSS, Bootstrap, etc) and clickable (JavaScript).
+
+## Do I need to install anything to write html?
+Theoretycally speaking - no, you don't,because you probably already have everything needed.
```
Comment (iteles) [inline README.md:19]: Tiny typo Theoretycally :arrow_right: Theoretically

File: README.md [hunk 4]
```diff
@@ -1,2 +1,346 @@
...
+You can write on anything(even Word) and save it in a .html extension.
+But truth is, if you want to make your life easier,
+you should install some aditional software - Notepad, Visual Studio Code, gedit/vim/nano, Atom, Sublime, etc.
```
Comment (iteles) [inline README.md:22]: A link to https://github.com/dwyl/dev-setup here would be a great addition!

File: README.md [hunk 5]
```diff
@@ -1,2 +1,346 @@
...
+</ul>
+
+Okay, so we write html in a text editor. Then how does our PC know it's a HTML file and not a poem or essay we've written? 
+Easy! HTML has it's own syntax( it's very close to the English language, don't worry) and it's own extension - **_.html_**.
+
+## HTML head and body
+
+Every html starts with the tag 
+``` html
+<!DOCTYPE html>
```
Comment (iteles) [inline README.md:43]: 👍 Did a little check on the accessibility to see if there was anything around doctype and this is perfect. 
https://www.paciellogroup.com/blog/2012/04/html-data-for-the-masses-doctype-usage/

File: README.md [hunk 6]
```diff
@@ -1,2 +1,346 @@
...
+<html> tag
+```
+
+The html tag has a closing tag:
+
+``` html 
+</html>
+```
+
+In between the html tags, our code looks just like a human - it starts with
```
Comment (iteles) [inline README.md:84]: 😄 🤸‍♂️

File: README.md [hunk 7]
```diff
@@ -1,2 +1,346 @@
...
+
+To write a sentence(or paragraph, word, number, etc), we simply put what we want to be shown
+on the screen between an opening and a closing tag.
+
+``` html 
+<p> Your text goes here </p>
+```
+ The paragraph tag should be used in the body. 
+
+<b> Extra knowledge: </b> HTML doesn't add new lines
```
Comment (iteles) [inline README.md:126]: 👍

File: README.md [hunk 8]
```diff
@@ -1,2 +1,346 @@
...
+when you press Enter in your editor. To add a new line, we use the 
+
+``` html 
+<br>
+```
+
+tag. **N.B!**The br tag has only an opening tag!!
```
Comment (iteles) [inline README.md:133]: It's actually known as a 'self-closing' tag to be strictly correct about it 😉 

I think we're also missing the closing `**` here to finish the bold portion of this section.

File: README.md [hunk 9]
```diff
@@ -1,2 +1,346 @@
...
+```
+
+## Links
+Now let's learn about links. For exammple, you want when a user clicks on a text, to open your other site.
+
+``` html
+<a href = "link_address_here"> The text the user should click here </a>
+```
+
+But have you noticed how some links open in the same tab and others for example open in a new tab. This is done with **the target attribute**.
```
Comment (iteles) [inline README.md:275]: Missing a question mark at the end of this sentence.

File: README.md [hunk 10]
```diff
@@ -1,2 +1,346 @@
...
+## Do I need to install anything to write html?
+Theoretycally speaking - no, you don't,because you probably already have everything needed.
+You can write on anything(even Word) and save it in a .html extension.
+But truth is, if you want to make your life easier,
+you should install some aditional software - Notepad, Visual Studio Code, gedit/vim/nano, Atom, Sublime, etc.
+There are billions of text editors that will help you out but the one you need is probably Notepad++.
+> https://notepad-plus-plus.org
+
+## What resources do I need?
+Most of the knowledge you'll need, will be in this guide. But in the IT world things are changing pretty fast. So make sure you educate yourself on new innovations, best practices and technologies. Make sure to check out our resources.md file where you will find some interesting guides, books and even more additional information.
```
Comment (iteles) [inline README.md:27]: Would be good to hyperlink in the resources file

File: syntax(done and to do).md [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+<li> Horizontal line
+<li> Comments
+<li> Simple text formatting tags
+<li> Headings
+<li> Images
+<li> Links
+<li> Lists
+<li> tables
+
+## TO DO:
```
Comment (iteles) [inline syntax(done and to do).md:14]: Let's get these into issues so anyone can pick them up and help add to the tutorial!

**Review by iteles — APPROVED** [review]
Wow @karinakozarova, this is absolutely incredible!! Sorry it's taken me so long to review as I have been away from my computer a lot this last week.

Thank you SO MUCH for this! Merging in as all the comments I've made are _tiny_ and can be dealt with in the next iteration. What we have here is too valuable not to share!

**Comment by karinakozarova** [comment]
Thanks, will fix it asap.

**Pull request merged**